### PR TITLE
Code-Server - VSCode editor for web

### PR DESCRIPTION
**Pull request by DatMayo**
### All Submissions:

* [X] Have you followed the guidelines in our Contributing document?
* [X] Have you checked to ensure there aren't other open [Pull Requests](../../pulls) for the same update/change?
* [X] Did you branch your changes and PR from that branch and not from your master branch?

### New Server Submissions:

1. [X] Does your submission pass tests (server is connectable)?
2. [x] Have you added the server to the main README.md?
3. [X] Have you added a unique README.md for the server you are adding?

### ToDo
* [x] Figure out, why the yaml file wont be updated
* [x] Add script into main README.md

### CODE REVIEW

**Comment by DatMayo** [comment]
Followup for PR https://github.com/parkervcp/eggs/pull/913
This PR is marked as draft until I (or someone else) can figure out, how to set values inside the yaml file

**Comment by DatMayo** [comment]
@parkervcp any chance this will be merged soon?

**Comment by parkervcp** [comment]
@DatMayo I swear I have an open code review here

**Comment by DatMayo** [comment]
I found a rly stupid bug, so I switch back to WIP

File: software/code-server/egg-code--server.json [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+            "description": "Passwort to login to your Code-Server instance",
+            "env_variable": "PASSWORD",
+            "default_value": "changeme",
+            "user_viewable": true,
+            "user_editable": true,
+            "rules": "required|string|max:32"
+        },
+        {
+            "name": "IP",
+            "description": "IP where the interface can be accessed",
```
[the remark this answers is not in the document]
Comment (DatMayo) [reply software/code-server/egg-code--server.json:41]: Jep, it was my mistake. I allready fixed that in my local version. I'll commit my changes this day

**Comment by DatMayo** [comment]
Ok, took me some testing, but its ready for review again :)

File: software/code-server/egg-code--server.json [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    },
+    "exported_at": "2021-01-27T07:26:08+01:00",
+    "name": "Code-Server",
+    "author": "mario.franze@gmail.com",
+    "description": "Run VS Code on any machine anywhere and access it in the browser.",
+    "features": null,
+    "images": [
+        "quay.io\/parkervcp\/pterodactyl-images:debian_nodejs-14"
+    ],
+    "startup": "sh .local\/lib\/code-server-3.8.0\/bin\/code-server",
```
Comment (elias-knodel) [inline software/code-server/egg-code--server.json:15]: If this PR is still "open": 
The startup script only works if you use version `3.8.0`. 
To fix this please change `sh .local\/lib\/code-server-3.8.0\/bin\/code-server` to `sh .local\/lib\/code-server-{{VERSION}}\/bin\/code-server`

Comment (DatMayo) [reply software/code-server/egg-code--server.json:15]: Good point, I'll change it within the next 24h.
Thx for your help <3

**Pull request merged**